lms/djangoapps/teams/templates/teams/teams.html

```diff
@@ -11,14 +11,18 @@
 
 <%include file="/courseware/course_navigation.html" args="active_page='teams'" />
 
-<div class="team-tab-content"></div>
+<div class="container">
+    <div class="course-wrapper">
+        <section class="course-content" id="course-content">
```

Comment:
That's to line up with the markup in #7859, specifically [here](https://github.com/edx/edx-platform/pull/7859/files#diff-8afc23486264ed446f06cadf2d0fec5fR132).